tests/draft7/ref.json

```diff
@@ -644,5 +644,69 @@
                 "valid": false
             }
         ]
+    },
+    {
+        "description": "order of evaluation: $id and $ref",
+        "schema": {
+            "$comment": "$id must be evaluated before $ref to get the proper $ref destination",
+            "$id": "/base/base.json",
+            "$ref": "int.json",
+            "$defs": {
+                "bigint": {
+                    "$comment": "canonical uri: /base/int.json",
+                    "$id": "int.json",
+                    "maximum": 10
+                },
+                "smallint": {
+                    "$comment": "canonical uri: /int.json",
+                    "$id": "/int.json",
+                    "maximum": 2
+                }
+            }
+        },
```

Comment:
This test is wrong. `$id` in the presence of `$ref` in draft7 will do nothing. consequently, attempts to resolve the ref to "/int.json" will fail.

It looks like I copied this test erroneously from draft2019-09, and it was never tested.